Spiral_Array/spiralarray.c

```diff
@@ -0,0 +1,101 @@
+#include<stdio.h>
```

Comment:
Please change the file to Spiral_Array/Spiral_Array.c

Reply:
I tried doing so...but it is showing file name already exists...
what should i do?